Components/BlockImageTextParallax/script.js

```diff
@@ -0,0 +1,51 @@
+import Rellax from 'rellax'
+import $ from 'jquery'
+
+const $window = $(window)
+
+let rellax, initialized
+
+class BlockImageTextParallax extends window.HTMLDivElement {
+  constructor (self) {
+    self = super(self)
+    self.$ = $(self)
+    return self
+  }
+
+  connectedCallback () {
+    initParallax()
+  }
+}
+
+function initParallax () {
```

Comment:
Functions should be class methods instead

Reply:
_ignored this one as agreed_